featuremanagement/_featuremanager.py

```diff
@@ -75,7 +80,9 @@ def get_variant(self, feature_flag_id, user_id, **kwargs):
         :rtype: Variant
         """
 
-    def get_variant(self, feature_flag_id, *args, **kwargs):
+    def get_variant(  # type: ignore
```

Comment:
Why do we need type ignore here?

Reply:
mypy currently throws an error saying `Overloaded function implementation does not accept all possible arguments of signature 1`. I'm trying to limit the size of each change as there is already a lot of them in this PR. If you look at the `mypy.ini` file I have a couple things set to false as they would require even more changes.

Also, I think this specific one is about the issue we have talked about before. I'll get to this in another PR.